nsv13/code/modules/overmap/fighters/_fighters.dm

```diff
@@ -403,6 +403,35 @@ Been a mess since 2018, we'll fix it someday (probably)
 						/obj/item/fighter_component/battery,
 						/obj/item/fighter_component/primary/cannon)
 
+/obj/structure/overmap/small_craft/combat/solgov
+	name = "Peregrine class attack fighter"
+	desc = "A Peregrine class attack fighter, solgov's only premiere fighter, mounting minature capital grade phasers and a tiny shield generator."
+	icon = 'nsv13/icons/overmap/new/solgov/playablefighter.dmi'
+	armor = list("melee" = 60, "bullet" = 60, "laser" = 60, "energy" = 30, "bomb" = 30, "bio" = 100, "rad" = 90, "fire" = 90, "acid" = 80, "overmap_light" = 5, "overmap_medium" = 0, "overmap_heavy" = 10) // no armor to speak of, tissue paper
```

Comment:
Given the your average fighters overmap health, heavy armor on standard fighter is basically irrelevant. The comment about "no armor to speak of" is not required, nor is this notably more fragile than any other fighter with this armor set.

Reply:
the light fighter has 200 integrity compared to this one's 150 total, in addition to lower armor